src/functions-and-arrays.js

```diff
@@ -29,9 +90,39 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(wordsUnique){
+  var uniqueArray = [];
+
+  if(wordsUnique.length == 0){
+    return null; 
+  }
+
+  for(i=0; i < wordsUnique.length; i++){
```

Comment:
Please use a variable keyword for declaring "i".

https://stackoverflow.com/questions/5717126/var-or-no-var-in-javascripts-for-in-loop